### ISSUE TITLE
Hooks are broken on 9.1.4

### ISSUE THREAD

**Issue by hedger** [frida/frida-java#15]
I can't hook any method on latest frida.

`> frida -U com.android.terminal -l java.js`
```
Attaching...
Loaded.
Error: expected a pointer
    at i (frida/node_modules/frida-java/lib/android.js:315)
    at frida/node_modules/frida-java/lib/android.js:301
    at frida/node_modules/frida-java/lib/vm.js:35
    at o (frida/node_modules/frida-java/lib/android.js:311)
    at apply (native)
    at frida/node_modules/frida-java/lib/android.js:368
    at replaceArtImplementation (frida/node_modules/frida-java/lib/class-factory.js:1064)
    at frida/node_modules/frida-java/lib/class-factory.js:847
    at [anon] (repl1.js:24)
    [...]
```

java.js:
```
function onLoad() {
    console.log("Loaded.")

    Java.perform( function() { 
      var URL = Java.use('java.net.URL');
      URL.getHost.implementation = function () {
        console.log("inside hook!");
        return this.getHost();
      };
    });
}
onLoad();
```

**Comment by oleavr** [comment]
Which Android OS version is this on?

**Comment by hedger** [comment]
6.0.1, CM13.1.5

**Comment by oleavr** [comment]
Fixed in frida-java 1.0.11, bundled with Frida 9.1.6.

Nougat is also supported now, although frida-core will require a libsepol upgrade and SELinux rule tweaks to run there – @akiannillo is working on this.

**Comment by hedger** [comment]
Thanks!
